### PR TITLE
frontend design diagram

### PR DESCRIPTION
**Pull request by namframgia**

### CODE REVIEW

**Comment by namframgia** [comment]
https://edu-redmine.framgia.vn/issues/4265

File: app/Http/Resources/WorkspaceResource.php [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    /**
+     * Transform the resource into an array.
+     *
+     * @param  \Illuminate\Http\Request  $request
+     * @return array
+     */
+    public function toArray($request)
+    {
+        return [
+            'id' => $this->id,
```
Comment (framgiaciv3) [inline app/Http/Resources/WorkspaceResource.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Strings.DoubleQuoteUsage.NotRequired</br>**Severity**: error</br>**File**: app/Http/Resources/WorkspaceResource.php</br>**Line**: 21</br>String "" does not require double quotes; use single quotes instead

File: resources/js/design_without_diagram.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+        $('.cell-selected').attr('data-name', name);
+        $('.cell-selected').attr('class', 'seat-cell area-selected');
+    }
+
+    $(document).on('click', '.default-areas', function() {
+        let color;
+        let name;
+        if ($(this).hasClass('door')) {
+            color = '#757575';
+            name = 'door';
```
Comment (BunnyPi04) [inline resources/js/design_without_diagram.js:95]: ko có i18n à?

Comment (namframgia) [reply resources/js/design_without_diagram.js:95]: cái này chỉ là đặt tên cho thẻ thôi mà chị

**Pull request merged**